app/hooks/apps.ts

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {useCallback} from 'react';
+import {useIntl} from 'react-intl';
+
+import {handleBindingClick} from '@actions/remote/apps';
+import {handleGotoLocation} from '@actions/remote/command';
+import {AppCallResponseTypes} from '@app/constants/apps';
```

Comment:
In general, we try to avoid `@app/` imports. It would be great if you could change these imports like `@app/XXX...` to `@XXX...`.